src/shared/configUtils.ts

```diff
@@ -7,12 +7,38 @@
 
 import { Workspace } from '@lwc/lwc-dev-server';
 import { CryptoUtils, SSLCertificateData } from '@salesforce/lwc-dev-mobile-core';
-import { Config, ConfigAggregator } from '@salesforce/core';
+import { Config, ConfigAggregator, Connection } from '@salesforce/core';
 import configMeta, { ConfigVars, SerializedSSLCertificateData } from './../configMeta.js';
 
+// eslint-disable-next-line @typescript-eslint/consistent-type-definitions
```

Comment:
Meh, just make it a `type` then. I personally think this rule is a bit too heavy-handed on style choices, but we can use a type interchangeably here, so may as well go with it.